master/static/migrations/20210511152412_description-to-name.up.sql

```diff
@@ -0,0 +1,5 @@
+UPDATE experiments
```

Comment:
whether we're shimming in a migrations or shimming on query, i dont think we should be shimming in the database going forward, but this is probably fine for just this.

Reply:
hmm what would be some alternatives?